test/Microsoft.Azure.Devices.Edge.Test.Common/EdgeDevice.cs

```diff
@@ -43,7 +44,7 @@ public class EdgeDevice
                 iotHub.Hostname);
```

Comment:
Curious - Why is CreateIdentity interesting to profile?

Reply:
It really isn't, I think I did it for consistency since it's an async call that could technically timeout.